sensor_msgs/PointCloud2 cloud
sensor_msgs/PointCloud2 nrm_cloud

sensor_msgs/PointCloud2 objs_cloud
sensor_msgs/PointCloud2 objs_nrm
sensor_msgs/PointCloud2 objs_uv

sensor_msgs/PointCloud2 table_cloud
sensor_msgs/PointCloud2 table_nrm
sensor_msgs/PointCloud2 table_uv
float64[] table_params

sensor_msgs/PointCloud2 bg_cloud
sensor_msgs/PointCloud2 bg_nrm
sensor_msgs/PointCloud2 bg_uv

sensor_msgs/PointCloud2 objs_motion_cloud
sensor_msgs/PointCloud2 objs_motion_nrm
sensor_msgs/PointCloud2 objs_motion_uv

sensor_msgs/Image objs_rgb
sensor_msgs/Image table_rgb
sensor_msgs/Image bg_rgb

sensor_msgs/Image objs_mask
sensor_msgs/Image table_mask
sensor_msgs/Image bg_mask
sensor_msgs/Image motion_mask